srcs/main.c

```diff
@@ -6,11 +6,18 @@
 /*   By: arthurbeznik <arthurbeznik@student.coda      +#+                     */
 /*                                                   +#+                      */
 /*   Created: 2023/02/12 15:05:10 by arthurbezni   #+#    #+#                 */
-/*   Updated: 2023/02/12 15:05:28 by arthurbezni   ########   odam.nl         */
+/*   Updated: 2023/02/12 17:38:06 by abeznik       ########   odam.nl         */
 /*                                                                            */
 /* ************************************************************************** */
 
+#include "../includes/cub3D.h"
+
 int	main(int argc, char *argv[])
 {
+	if (parser(argc, argv))
```

Comment:
Better true or false return from parser. right now the statement doesn't make sense, if parser is true then return error.

Reply:
i hate bool but i'll change it to bool

Reply:
I mean we can also customize our own codes if you really hate bools but in way that's logical